projects/WTI-API/src/main/services/ClarificationService.java

```diff
@@ -17,10 +17,24 @@ public ClarificationService(String pId, WTIWebsocket socket) {
 		this.teamId = pId;
 		client = socket;
 	}
-
+	
 	@Override
 	public void clarificationAdded(IClarification arg0) {
-		
+        //This method is used by two (or more?) methods.
+        //Whenever an announcement that this user should receive is made this method is called
+        //Also whenever the team sends a clarification to be answered by the judges.
+        if ( !arg0.isAnswered()) {  //case when the team sends a clarification to be answered.
```

Comment:
It would be nice if the default template parameter name ("arg0") was changed to something more meaningful, like "clar" or "clarification".

Reply:
If my answer is satisfactory, I will close this.

Reply:
Keep in mind this was originally written by students and didn't go through the vigorous review process we do for PC2.  I don't thing we have to change all the existing code, but there is no sense in propagating bad practices by putting good code after bad.  I think we should change NEW code to be as readable as possible.  That is, I would change `arg0` to something meaningful, eg. `clar `or `newClar`

Reply:
Alright.